doc/external-postgres.md

```diff
@@ -0,0 +1,57 @@
+[NooBaa Operator](../README.md) /
+
+# External Postgresql DB support
+
+As part of a growing demand on supporting external Postgresql DB, we are now supporting a new basic option to use noobaa with an externally provided Postgresql DB. First, you will need to create a new empty DB with the needed collation by noobaa:
+1. Validate you have a working Postgres that can be accessed from external pods.
+2. Create a new DB that will be used by noobaa to save its metadata. Using the following SQL command:
+```sql
+CREATE DATABASE nbcore WITH LC_COLLATE = ‘C’ TEMPLATE template0;
```

Comment:
is this step really necessary?

https://github.com/noobaa/noobaa-core/blob/6281591bd5b7bc6142446666f570f01b3bcbf61c/src/util/postgres_client.js#L1371-L1383

Reply:
We currently not creating a database on our own (the DB image creates the DB at first run), and the user probably won't give us admin credentials that can create new databases (I think). So I think it's kind of a must.
@dannyzaken WDYT?